README.md

```diff
@@ -312,6 +312,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [sharkdp/bat](https://github.com/sharkdp/bat) — A cat(1) clone with wings. [<img src="https://api.travis-ci.org/sharkdp/bat.svg?branch=master">](https://travis-ci.org/sharkdp/bat)
 * [sharkdp/fd](https://github.com/sharkdp/fd) — A simple, fast and user-friendly alternative to find. [![Build Status](https://api.travis-ci.org/sharkdp/fd.svg?branch=master)](https://travis-ci.org/sharkdp/fd)
 * [sitkevij/hex](https://github.com/sitkevij/hex) — A colorized hexdump terminal utility. [<img src="https://api.travis-ci.org/sitkevij/hex.svg?branch=master">](https://travis-ci.org/sitkevij/hex)
+* [slai11/goto](https://github.com/slai11/goto) — A simple and user-friendly way to jump to your indexed directories. ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/slai11/goto/release) 
```

Comment:
Can you make the CI badge link to the CI builds please?

Reply:
Sure, i've linked it to the actions. I referred to some example in the existing readme.